src/axis.ts

```diff
@@ -86,6 +86,11 @@ export interface Axis extends BaseAxisNoSignals, Guide {
    */
   tickStep?: number;
 
+  /**
+   * A type of axis format to be forced to.
+   */
+  formatType?: 'number' | 'time';
```

Comment:
Better add this to `Guide`, which both `Axis` and `Legend` extend from.  
Please also add it to `TextFieldDef`.